src/Raven.Client/Documents/Session/AsyncDocumentSession.RawQuery.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Raven.Client.Documents.Indexes;
+
+namespace Raven.Client.Documents.Session
+{
+    public partial class AsyncDocumentSession
+    {
+        public IAsyncRawDocumentQuery<T> AsyncRawQuery<T>(string query, string indexName = null)
```

Comment:
query needs to define index, so we can remove 'indexName' I think?

Reply:
Yes, fixed